MaxText/layers/linears.py

```diff
@@ -30,9 +31,13 @@
 from jax.ad_checkpoint import checkpoint_name
 from jax.experimental import shard_map
 import max_logging
+import max_utils
+from aqt.jax.v2 import aqt_tensor
 
 try:
-  from jax.experimental.pallas.ops.tpu import megablox as mblx
+  # from jax.experimental.pallas.ops.tpu import megablox as mblx
```

Comment:
We previously add this comment as it throws error when running on GPU. Now we moved this into MaxText, the ImportError will not be triggered anymore. We could directly put `from kernels import megablox as mblx`.